### ISSUE TITLE
Sphinx documentation generation does not work

### ISSUE THREAD

**Issue by DaveStrickland** [DaveStrickland/AstroPhotography#15]
Attempting to generate the sphinx documentation fails.

The may be a configuration issue and/or a code documentation style issue. 

We need to pick a format of code documatation that works (preferably `astropy`, or maybe `numpy`), convert all doc strings to it, and get the documentation working.

**Comment by DaveStrickland** [comment]
Actively working this issue now.

**Comment by DaveStrickland** [comment]
After a big pause, I've finally gotten around to working on this.

It requires some reorganization of the package. To build the documentation:
```bash
#git pull origin
cd doc
make html
# documentation index file will be in $(pwd)/_build/html/index.html
```

**Comment by DaveStrickland** [comment]
Resolved by bad2479
